spec/features/staff/users_can_view_reports_spec.rb

```diff
@@ -374,6 +374,18 @@
         end
       end
 
+      scenario "they see helpful content about uploading acutals spend data and a link to the template on the actuals tab" do
+        report = create(:report, :active, organisation: delivery_partner_user.organisation)
+
+        visit report_actuals_path(report)
+
+        expect(page.html).to include t("tabs.transactions.upload.copy_html")
+        expect(page).to have_link t("page_content.transactions.button.download_template"),
+          href: report_transaction_upload_path(report, format: :csv)
+        expect(page).to have_link "guidance in the help centre (opens in new tab)",
+          href: "https://beisodahelp.zendesk.com/hc/en-gb/articles/1500005601882-Downloading-the-Actuals-Template-in-order-to-Bulk-Upload"
+      end
```

Comment:
This is really easy to understand. 

If I had more, and more complicated, features to describe on this page I might go for a structure like this, using helper methods to i) express the user benefits clearly and ii) get the incidental implementation details out of the way:

```
scenario "views report actuals" do
   report = create(:report, :active, organisation: delivery_partner_user.organisation)

  visit report_actuals_path(report)

  expect_to_see_guidance_on_uploading_my_actuals
  expect_to_be_able_to_download_a_csv_template
  expect_to_have_a_link_to_actuals_content_in_the_help_centre
 
end
```

Reply:
Very nice, I'll keep that in mind, I do wonder if there is a limit somewhere for this, but as you say, it is probably on something more complicated - perhaps that is where view specs might come in to play?

Reply:
yes, If you had 3 times as many Capybara expectations here then it might get hard to read and be worth breaking down into helpers.

For me it's all about pushing the intention of the code to the forefront of our minds. That makes it easier to see if the implementation is really matching the stated purpose as well as it might...

Of course I am a big fan of "cucumber" which is designed to this style of testing...

Reply:
Yeah, one day I might have some mental capacity to look at cucumber, does eLinks use it - I have a memory of seeing it there? I still feel I have a long ways to go with rspec yet!? 😆